tensorflow/core/kernels/data/map_dataset_op.cc

```diff
@@ -27,6 +28,8 @@ namespace {
 // See documentation in ../../ops/dataset_ops.cc for a high-level
 // description of the following op.
 
+using namespace tensorflow::data::name_utils;
```

Comment:
remove this and instead use `name_utils::IteratorPrefix` and `name_utils::DatasetDebugString` ... importing entire namespace "pollutes" the context with all of its symbols

Reply:
Done